src/app/recents/recents.component.css

```diff
@@ -0,0 +1,7 @@
+.recent-title {
+    margin-left: 6rem;
+}
+
+.recent-tabs {
+    padding-left: 6rem;
```

Comment:
This should be "margin-left" since you just want to shift the div over, you don't want the div to be larger.